src/web/screens/manageParticipants.tsx

```diff
@@ -42,15 +45,19 @@ function ManageParticipants() {
       </p>
       <Suspense fallback={<Loading />}>
         <Await resolve={data.results}>
-          {([participantRequests, participantTypes]: [
+          {([participantRequests, participantApproved, participantTypes]: [
             ParticipantRequestDTO[],
+            ParticipantDTO[],
             ParticipantTypeDTO[]
           ]) => (
-            <ParticipantRequestsTable
-              participantRequests={participantRequests}
-              participantTypes={participantTypes}
-              onApprove={handleApproveParticipantRequest}
-            />
+            <div>
```

Comment:
nit: I'd recommend just using `<>` here instead of a `<div>`. I think it ends up being a `<div>` in the DOM anyway but it reads a bit cleaner. You'd want to use `<div>` if it was being used for styling.